packages/framework-core/src/decorators/global-event-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { EventHandlerInterface, BoosterConfig } from '@boostercloud/framework-types'
+import { Booster } from '../booster'
+
+export const GLOBAL_EVENT_HANDLER = 'GLOBAL_EVENT_HANDLER'
```

Comment:
To minimize the chances of collision with a user defined event handler, I would prefix this with `BOOSTER`: 'BOOSTER_GLOBAL_EVENT_HANDLER'

Also, and this is being nitpicking, I would add an S to both the var name and value, because we are storing multiple event handlers